scripts/tasks/perf-test.js

```diff
@@ -0,0 +1,88 @@
+const urlFromDeployJob = `http://fabricweb.z5.web.core.windows.net/pr-deploy-site/${process.env.BUILD_SOURCEBRANCH}/perf-test/`;
```

Comment:
Any chance we can have a localhost version of this for devs who need to verify the local changes to have the same perf as master? Perhaps not in this PR, maybe add a todo?

```suggestion
// TODO: add the ability to run this perf-test against a non PR deployed site
const urlFromDeployJob = `http://fabricweb.z5.web.core.windows.net/pr-deploy-site/${process.env.BUILD_SOURCEBRANCH}/perf-test/`;
```